netmaster/objApi/objapi_test.go

```diff
@@ -1002,6 +1002,10 @@ func TestNetworkAddDelete(t *testing.T) {
 	verifyNetworkState(t, "default", "contiv", "data", "vxlan", "10.1.1.1", "10.1.1.254", 16, 1, 1, "", "", 0)
 	checkDeleteNetwork(t, false, "default", "contiv")
 
+	// vxlan network with VNI higher than 4K
+	checkCreateNetwork(t, false, "default", "contiv", "", "vxlan", "10.1.1.1/24", "10.1.1.254", 8000, "", "")
+	checkDeleteNetwork(t, false, "default", "contiv")
+
```

Comment:
It looks like this commit only tests for https://github.com/contiv/netplugin/issues/508 but doesn't fix it.  I'm not seeing the issue though, so probably some other commit fixed it?